### PR TITLE
Add template: Darktrace vSensor standalone

### PR DESCRIPTION
**Pull request by menev-dt**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Review by bmoore-msft — CHANGES_REQUESTED** [review]
@menev-dt - can you move this to the application-workloads folder?

**Comment by menev-dt** [comment]
> @menev-dt - can you move this to the application-workloads folder?
@bmoore-msft - done, thanks.

File: application-workloads/darktrace-vsensor-standalone/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+        },
+        "virtualMachineSize": {
+            "defaultValue": "Standard_D2s_v3",
+            "type": "String",
+            "metadata": {
+                "description": "The VM size. Check the Darktrace customer portal for more information about the vSensor Virtual Hardware requirements."
+            }
+        },
+        "adminUsername": {
+            "defaultValue": "ubuntu",
```
Comment (bmoore-msft) [inline application-workloads/darktrace-vsensor-standalone/azuredeploy.json:74]: remove defaultValue (increases surface area)

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: application-workloads/darktrace-vsensor-standalone/azuredeploy.json (hunk 1 above)
Comment (menev-dt) [reply application-workloads/darktrace-vsensor-standalone/azuredeploy.json]: defaultValue removed.

May I ask what 'increases surface area' means in this context?

Comment (bmoore-msft) [reply application-workloads/darktrace-vsensor-standalone/azuredeploy.json]: Users tend to use defaultValues and having a username be known to an attacker gives them more to work with...

**Pull request merged**